examples/README.md

```diff
@@ -0,0 +1,23 @@
+# Examples
+
+To run an example `npm run start:example -- "name of example"`. Eg to run the basic example at `./basic.js`:
+
+```shell script
+$ npm run start:example -- basic
+```
```

Comment:
I recommend listing the available examples here. I know they're listed out somewhat on the EXAMPLES.md page, but would be nice to have a concise reference somewhere here.

Reply:
Actually I can see how the "basic" example would be ok, in that the name is the same as the filename. Still, interested in your thoughts.

Reply:
It's probably one of those things that would go out of date, also when you're looking at the `examples/README.md` you'll usually have the latest list above:

![image](https://user-images.githubusercontent.com/1299658/91168468-99214500-e6cd-11ea-8c87-8a7d7607c3b6.png)